src/client/app/containers/app.ts

```diff
@@ -198,6 +205,8 @@ export class AppComponent {
                     if (confirmationAlertIfAny !== this.strings.cancelButtonLabel) {
                         this._store.dispatch(new GitHub.SharePublicGistAction(this.snippet));
                     }
+
+                    this.isDisabled = false;
```

Comment:
Is setting this.isDisabled necessary anywhere other than in the this._store.select(fromRoot.getSharing).subscribe observer?